include/alpaka/mem/buf/cpu/CpuBuf.hpp

```diff
@@ -0,0 +1,17 @@
+/* Copyright 2025 Anton Reinhard
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+#include "alpaka/mem/buf/cpu/ConstBufCpu.hpp"
+#include "alpaka/mem/buf/cpu/MutBufCpu.hpp"
+
+namespace alpaka
+{
+    template<typename TElem, typename TDim, typename TIdx>
+    using BufCpu = MutBufCpu<ConstBufCpu, alpaka::detail::BufCpuImpl<TElem, TDim, TIdx>, DevCpu, TElem, TDim, TIdx>;
```

Comment:
In fact, what is the point of having a `MutBufCpu` class and a `BufCpu` alias ?
Why not simply have a `BufCpu` class ?

Reply:
The point was to hide the additional template arguments I introduced from users of the class. But I can try to get rid of them by deriving them inside the class directly, I think it should be possible.

Reply:
Mhm, I see.

Reply:
We should discuss this via VC or directly. I still think, we can make it easier. For me, I don't understand why we need the template parameter `ConstBufCpu` **and** `alpaka::detail::BufCpuImpl` **and** `DevCpu`.

I think, if I understand the code, we can find a more understandable solution, maybe by inheritance, maybe with traits and concepts or something else.

Reply:
Yes I already noticed they are not needed anymore at all, it was leftover from my previous more generic attempt at an implementation. Please wait a bit until I can revise what I have, then we can talk about that.

Reply:
Please take a look at ef24409ef8de95123204c9be82b4d48a568eebea